docker/images/ApiZapScan/security.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+echo ${TEST_URL}
+zap-api-scan.py -t ${TEST_URL} -f openapi -S -d -u ${SecurityRules} -P 80 -l FAIL
```

Comment:
Possible Small refactoring is to change TEST_URL to ENDPOINT_URL like how we have refactored in other commands...